x-pack/plugins/triggers_actions_ui/public/application/sections/alert_form/alert_add.tsx

```diff
@@ -91,16 +97,35 @@ const AlertAdd = ({
     }
   }, [alertTypeId]);
 
-  const closeFlyout = useCallback(() => {
-    setAlert(initialAlert);
-    onClose();
-  }, [initialAlert, onClose]);
+  useEffect(() => {
+    if (isEmpty(alert.params) && !isEmpty(initialAlertParams)) {
+      // alert params are explicitly cleared when the alert type is cleared
+      // clear the "initial" in order to params in order to capture the
+      // default when a new alert type is selected
```

Comment:
**nit:** the comment is a bit hard to read. I think the suggestion below was the intention of the comment?
```suggestion
      // alert params are explicitly cleared when the alert type is cleared.
      // clear the "initial" params in order to capture the
      // default when a new alert type is selected
```

Reply:
Good catch! Updated.